src/styles/variables/_variables.scss

```diff
@@ -0,0 +1,11 @@
+$cell-size: 100px;
+$cell-gap: 1px;
+$calendar-width: $cell-size * 7 + $cell-gap * 6;
+$days:
+  "mon" 0,
+  "tue" calc(#{$cell-size} + #{$cell-gap}),
+  "wed" calc(2 * (#{$cell-size} + #{$cell-gap})),
+  "thu" calc(3 * (#{$cell-size} + #{$cell-gap})),
+  "fri" calc(4 * (#{$cell-size} + #{$cell-gap})),
+  "sat" calc(5 * (#{$cell-size} + #{$cell-gap})),
+  "sun" calc(6 * (#{$cell-size} + #{$cell-gap}));
```

Comment:
You often use similar math combinations here, you can create a function for this moment or someone variable.

Reply:
agree, do it

Reply:
Please, create variable for this operation cell-size * cell-gap
Also, you can create a fucntion